databricks_cli/unity_catalog/perms_cli.py

```diff
@@ -0,0 +1,134 @@
+@debug_option
+@profile_option
+@eat_exceptions
+@provide_api_client
+def get_permissions_cli(api_client, catalog, schema, table, storage_credential,
+                        external_location):
+    """
+    Get permissions on a securable.
+
+    Calls the 'getPermissions' RPC endpoint of the Unity Catalog service.
+    Returns PermissionsList for the requested securable.
+
+    """
+    sec_type, sec_name = _get_perm_securable_name_and_type(catalog, schema, table,
+                                                           storage_credential, external_location)
+
+    perm_json = UnityCatalogApi(api_client).get_permissions(sec_type, sec_name)
+    click.echo(mc_pretty_format(perm_json))
+
+
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help='update permissions on a securable.')
+@click.option('--catalog', cls=OneOfOption, default=None,
+              one_of=PERMISSIONS_OBJ_TYPES,
+              help='Name of catalog of interest')
+@click.option('--schema', cls=OneOfOption, default=None,
+              one_of=PERMISSIONS_OBJ_TYPES,
+              help='Full name of schema of interest')
+@click.option('--table', cls=OneOfOption, default=None,
+              one_of=PERMISSIONS_OBJ_TYPES,
+              help='Full name of table of interest')
+@click.option('--storage-credential', cls=OneOfOption, default=None,
+              one_of=PERMISSIONS_OBJ_TYPES,
+              help='Name of the storage credential of interest')
+@click.option('--external-location', cls=OneOfOption, default=None,
+              one_of=PERMISSIONS_OBJ_TYPES,
+              help='Name of the external location of interest')
+@click.option('--json-file', default=None, type=click.Path(),
+              help='File containing JSON of permissions change to PATCH.')
+@click.option('--json', default=None, type=JsonClickType(),
+              help=JsonClickType.help('/api/2.0/unity-catalog/permissions'))
+@debug_option
+@profile_option
+@eat_exceptions
+@provide_api_client
+def update_permissions_cli(api_client, catalog, schema, table, storage_credential,
+                           external_location, json_file, json):
+    """
+    Update permissions on a securable.
+
+    Calls the 'updatePermissions' RPC endpoint of the Unity Catalog service.
+    Returns updated PermissionsList for the requested securable.
+
+    """
+    sec_type, sec_name = _get_perm_securable_name_and_type(catalog, schema, table,
+                                                           storage_credential, external_location)
+
+    json_cli_base(json_file, json,
+                  lambda json: UnityCatalogApi(api_client).update_permissions(sec_type, sec_name,
+                                                                              json),
+                  encode_utf8=True)
+
+
+def register_perms_commands(cmd_group):
+    cmd_group.add_command(get_permissions_cli, name='get-permissions')
+    cmd_group.add_command(update_permissions_cli, name='update-permissions')
```

Comment:
please make command naming consistent with the rest of the CLI (see comments for other command groups). command group should be `data-permissions` or `uc-permissions` (or any other), that would not conflict with the one in front of Generic Permissions API

Reply:
Addressed in #471. The commands are under `databricks unity-catalog permissions`.